charts/src/main/java/hu/ma/charts/legend/HorizontalLegend.kt

```diff
@@ -5,35 +5,32 @@ import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.Row
 import androidx.compose.foundation.layout.Spacer
 import androidx.compose.foundation.layout.requiredSize
+import androidx.compose.foundation.layout.wrapContentSize
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.unit.dp
-import androidx.compose.ui.util.fastForEach
-import com.google.accompanist.flowlayout.FlowRow
+import androidx.compose.ui.util.fastForEachIndexed
 import hu.ma.charts.internal.DefaultText
 
 @Composable
 fun DrawHorizontalLegend(
   legendEntries: List<LegendEntry>,
   text: @Composable (item: LegendEntry) -> Unit = { DefaultText(text = it.text) },
 ) {
-  FlowRow(
-    mainAxisSpacing = 16.dp,
-    crossAxisSpacing = 8.dp,
-  ) {
```

Comment:
This change breaks some of the sane-defaults expectations for the bars chart. What's the reasoning for this change instead of using a custom legend?

Reply:
FlowRow is was deprecated in alpha10 (https://developer.android.com/jetpack/androidx/releases/compose-foundation#1.0.0-alpha10), and I had trouble getting it to fit well with LineChart. Thought i had verified it was still working for the other charts, but I will revert and create a custom legend for LineChart instead.

Reply:
This one is coming from [accompanist/flowlayout](https://google.github.io/accompanist/flowlayout/) so it's fine to use

Reply:
Ah, my bad. Will fix it.